vspace/src/main/java/edu/asu/diging/vspace/core/model/SpaceOverview.java

```diff
@@ -0,0 +1,86 @@
+package edu.asu.diging.vspace.core.model;
```

Comment:
this class is only used to pass data to the front end, isn't it? It shouldn't be in the core model package.